examples/nfct_test.rs

```diff
@@ -0,0 +1,170 @@
+#![no_std]
+
+extern crate alloc;
+extern crate lang_items;
+extern crate libtock_drivers;
+
+use core::fmt::Write;
+use libtock_core::result::CommandError;
+use libtock_drivers::console::Console;
+use libtock_drivers::nfc::NfcTag;
+use libtock_drivers::nfc::RecvOp;
+use libtock_drivers::result::FlexUnwrap;
+use libtock_drivers::result::TockError;
+use libtock_drivers::timer;
+use libtock_drivers::timer::Timer;
+use libtock_drivers::timer::Timestamp;
+
+#[allow(dead_code)]
+/// Helper function to write on console the received packet.
+fn print_rx_buffer(buf: &mut [u8], amount: usize) {
```

Comment:
nit: I just noticed here, that `amount` is an usual name for a buffer size. It's `size` or `length` most of the time.

Reply:
`amount` reflects the amount of data received and not the whole size of the buffer.